easybuild/tools/github.py

```diff
@@ -1000,7 +1003,9 @@ def check_github():
 
     # GitHub user
     print_msg("* GitHub user...", log=_log, prefix=False, newline=False)
-    github_user = build_option('github_user')
+    github_user = github_account = build_option('github_user')
+    if build_option('github_org'):
+        github_account = build_option('github_org')
```

Comment:
@timeu since this logic is used in multiple places, wouldn't it be better to define a small function that determines what to use as upstream?